cross-chain/base/external/base/BaseWormholeTBTC.json

```diff
@@ -0,0 +1,3 @@
+{
+  "address": "0x9EE95E6Bd1B3C5740F105d6fb06b8BDeF64Eec70"
```

Comment:
NB: I could not double-check this address as I don't know where it comes from. Keep this in mind during deployment

Reply:
Sure, this was confirmed by @dimpar on Discord. The address comes from the wrapped tBTC token registration on Base (tx: https://basescan.org/tx/0x82af9e68b6be7db6b89cfb6e09368f524e04a317c938e367047f6467484e47a3).